CRITICAL - 2025-05-16 08:40:04 --> CodeIgniter\Exceptions\FrameworkException: The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 252.
 1 SYSTEMPATH\CodeIgniter.php(252): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(214): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-05-16 08:40:05 --> ErrorException: Uncaught Error: Class "Locale" not found in C:\Proyectos\reservalo2.0\system\Helpers\number_helper.php:137
Stack trace:
#0 C:\Proyectos\reservalo2.0\system\Helpers\number_helper.php(56): format_number(4.0, 2, NULL, Array)
#1 C:\Proyectos\reservalo2.0\system\Debug\BaseExceptionHandler.php(152): number_to_size(4.0, 2)
#2 C:\Proyectos\reservalo2.0\app\Views\errors\html\error_exception.php(384): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(4194304)
#3 C:\Proyectos\reservalo2.0\system\Debug\BaseExceptionHandler.php(258): include('C:\\Proyectos\\re...')
#4 C:\Proyectos\reservalo2.0\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\Proyectos\reservalo2.0\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\FrameworkException), 500, 'C:\\Proyectos\\re...')
#6 C:\Proyectos\reservalo2.0\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\FrameworkException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\FrameworkException))
#8 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\FrameworkException
The framework needs the following extension(s) installed and loaded: intl.
#0 C:\Proyectos\reservalo2.0\system\CodeIgniter.php(252): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\Proyectos\reservalo2.0\system\CodeIgniter.php(214): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\Proyectos\reservalo2.0\index.php(67): CodeIgniter\CodeIgniter->initialize()
#3 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 137.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2025-05-16 06:40:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-05-16 07:21:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-05-16 07:49:46 --> InvalidArgumentException: BDReservalo2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 SYSTEMPATH\Model.php(153): CodeIgniter\Database\Config::connect('BDReservalo2')
 2 APPPATH\Controllers\Login.php(25): CodeIgniter\Model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->registrarse()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-05-16 07:49:49 --> InvalidArgumentException: BDReservalo2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 SYSTEMPATH\Model.php(153): CodeIgniter\Database\Config::connect('BDReservalo2')
 2 APPPATH\Controllers\Login.php(25): CodeIgniter\Model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->registrarse()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-05-16 08:18:54 --> InvalidArgumentException: BDReservalo2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 SYSTEMPATH\Model.php(153): CodeIgniter\Database\Config::connect('BDReservalo2')
 2 APPPATH\Controllers\Login.php(25): CodeIgniter\Model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->registrarse()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-05-16 08:19:32 --> InvalidArgumentException: BDReservalo2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 SYSTEMPATH\Model.php(153): CodeIgniter\Database\Config::connect('BDReservalo2')
 2 APPPATH\Controllers\Login.php(25): CodeIgniter\Model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->registrarse()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2025-05-16 11:06:11 --> mysqli_sql_exception: Unknown column 'role' in 'field list' in C:\Proyectos\reservalo2.0\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Proyectos\reservalo2.0\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\Proyectos\reservalo2.0\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\Proyectos\reservalo2.0\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\Proyectos\reservalo2.0\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Proyectos\reservalo2.0\app\Models\loginModel.php(47): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\Proyectos\reservalo2.0\app\Controllers\Login.php(69): App\Models\loginModel->registrar('dani.prueba@gma...', 'XXZKQD24.')
#6 C:\Proyectos\reservalo2.0\system\CodeIgniter.php(941): App\Controllers\Login->registrarse()
#7 C:\Proyectos\reservalo2.0\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Proyectos\reservalo2.0\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Proyectos\reservalo2.0\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2025-05-16 11:06:17 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'role' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`email`, `password`, `role`) VALUES (:email:, :password:, :role:)', [...], false)
 2 APPPATH\Models\loginModel.php(47): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Login.php(69): App\Models\loginModel->registrar('dani.prueba@gmail.com', 'XXZKQD24.')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->registrarse()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-05-16 11:24:49 --> ArgumentCountError: Too few arguments to function App\Models\loginModel::registrar(), 2 passed in C:\Proyectos\reservalo2.0\app\Controllers\Login.php on line 68 and exactly 3 expected
in APPPATH\Models\loginModel.php on line 28.
 1 APPPATH\Controllers\Login.php(68): App\Models\loginModel->registrar('dani.prueba@gmail.com', 'XXZKQD24.')
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->registrarse()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2025-05-16 11:45:50 --> mysqli_sql_exception: Unknown column 'rol' in 'field list' in C:\Proyectos\reservalo2.0\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Proyectos\reservalo2.0\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\Proyectos\reservalo2.0\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\Proyectos\reservalo2.0\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\Proyectos\reservalo2.0\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Proyectos\reservalo2.0\app\Models\loginModel.php(48): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\Proyectos\reservalo2.0\app\Controllers\Login.php(69): App\Models\loginModel->registrar('Daniel ', 'dani.prueba@gma...', 'XXZKQD24')
#6 C:\Proyectos\reservalo2.0\system\CodeIgniter.php(941): App\Controllers\Login->registrarse()
#7 C:\Proyectos\reservalo2.0\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Proyectos\reservalo2.0\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Proyectos\reservalo2.0\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2025-05-16 11:45:50 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'rol' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `rol`) VALUES (:nombre:, :email:, :password:, :rol:)', [...], false)
 2 APPPATH\Models\loginModel.php(48): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Login.php(69): App\Models\loginModel->registrar('Daniel ', 'dani.prueba@gmail.com', 'XXZKQD24')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->registrarse()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-05-16 11:45:50 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'rol' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `rol`) VALUES (\'Daniel \', \'dani.prueba@gmail.com\', \'f39426ccdb0ce5e981c826977de6f19aaf2da984\', 1)')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `rol`) VALUES (\'Daniel \', \'dani.prueba@gmail.com\', \'f39426ccdb0ce5e981c826977de6f19aaf2da984\', 1)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `rol`) VALUES (:nombre:, :email:, :password:, :rol:)', [...], false)
 4 APPPATH\Models\loginModel.php(48): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Login.php(69): App\Models\loginModel->registrar('Daniel ', 'dani.prueba@gmail.com', 'XXZKQD24')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->registrarse()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-05-16 11:45:50 --> [Caused by] mysqli_sql_exception: Unknown column 'rol' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `rol`) VALUES (\'Daniel \', \'dani.prueba@gmail.com\', \'f39426ccdb0ce5e981c826977de6f19aaf2da984\', 1)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `rol`) VALUES (\'Daniel \', \'dani.prueba@gmail.com\', \'f39426ccdb0ce5e981c826977de6f19aaf2da984\', 1)')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `rol`) VALUES (\'Daniel \', \'dani.prueba@gmail.com\', \'f39426ccdb0ce5e981c826977de6f19aaf2da984\', 1)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `rol`) VALUES (:nombre:, :email:, :password:, :rol:)', [...], false)
 5 APPPATH\Models\loginModel.php(48): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Login.php(69): App\Models\loginModel->registrar('Daniel ', 'dani.prueba@gmail.com', 'XXZKQD24')
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->registrarse()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2025-05-16 11:47:16 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`reservalo2`.`usuarios`, CONSTRAINT `fk_usuarios_roles_01` FOREIGN KEY (`id_rol`) REFERENCES `roles` (`id_rol`) ON DELETE NO ACTION ON UPDATE CASCADE) in C:\Proyectos\reservalo2.0\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\Proyectos\reservalo2.0\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\Proyectos\reservalo2.0\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\Proyectos\reservalo2.0\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\Proyectos\reservalo2.0\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Proyectos\reservalo2.0\app\Models\loginModel.php(48): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\Proyectos\reservalo2.0\app\Controllers\Login.php(69): App\Models\loginModel->registrar('Daniel', 'dani.prueba@gma...', 'XXZKQD24.')
#6 C:\Proyectos\reservalo2.0\system\CodeIgniter.php(941): App\Controllers\Login->registrarse()
#7 C:\Proyectos\reservalo2.0\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Proyectos\reservalo2.0\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Proyectos\reservalo2.0\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2025-05-16 11:47:16 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`reservalo2`.`usuarios`, CONSTRAINT `fk_usuarios_roles_01` FOREIGN KEY (`id_rol`) REFERENCES `roles` (`id_rol`) ON DELETE NO ACTION ON UPDATE CASCADE)
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `id_rol`) VALUES (:nombre:, :email:, :password:, :id_rol:)', [...], false)
 2 APPPATH\Models\loginModel.php(48): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Login.php(69): App\Models\loginModel->registrar('Daniel', 'dani.prueba@gmail.com', 'XXZKQD24.')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->registrarse()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-05-16 11:47:16 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`reservalo2`.`usuarios`, CONSTRAINT `fk_usuarios_roles_01` FOREIGN KEY (`id_rol`) REFERENCES `roles` (`id_rol`) ON DELETE NO ACTION ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `id_rol`) VALUES (\'Daniel\', \'dani.prueba@gmail.com\', \'6cd417576f4790d1de516e45340adacae98656c5\', 1)')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `id_rol`) VALUES (\'Daniel\', \'dani.prueba@gmail.com\', \'6cd417576f4790d1de516e45340adacae98656c5\', 1)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `id_rol`) VALUES (:nombre:, :email:, :password:, :id_rol:)', [...], false)
 4 APPPATH\Models\loginModel.php(48): CodeIgniter\Database\BaseBuilder->insert([...])
 5 APPPATH\Controllers\Login.php(69): App\Models\loginModel->registrar('Daniel', 'dani.prueba@gmail.com', 'XXZKQD24.')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->registrarse()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-05-16 11:47:16 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`reservalo2`.`usuarios`, CONSTRAINT `fk_usuarios_roles_01` FOREIGN KEY (`id_rol`) REFERENCES `roles` (`id_rol`) ON DELETE NO ACTION ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `id_rol`) VALUES (\'Daniel\', \'dani.prueba@gmail.com\', \'6cd417576f4790d1de516e45340adacae98656c5\', 1)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `id_rol`) VALUES (\'Daniel\', \'dani.prueba@gmail.com\', \'6cd417576f4790d1de516e45340adacae98656c5\', 1)')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `id_rol`) VALUES (\'Daniel\', \'dani.prueba@gmail.com\', \'6cd417576f4790d1de516e45340adacae98656c5\', 1)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `email`, `password`, `id_rol`) VALUES (:nombre:, :email:, :password:, :id_rol:)', [...], false)
 5 APPPATH\Models\loginModel.php(48): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Login.php(69): App\Models\loginModel->registrar('Daniel', 'dani.prueba@gmail.com', 'XXZKQD24.')
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->registrarse()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
